nengo_spinnaker/node_io/ethernet.py

```diff
@@ -100,6 +100,7 @@ def prepare(self, model, controller, netlist):
                 # Store this transmission parameters to (x, y, p) map
                 self._node_outgoing[node].append((
                     (transmission_params.pre_slice,
+                     transmission_params.slice_in,
                      transmission_params.function,
                      transmission_params.full_transform(slice_out=False)),
```

Comment:
I think you might be able to get away with just adding `slice_pre=False` to this call.